### ISSUE TITLE
The behavior of tensor.grad after backpropgate.

### ISSUE THREAD

**Issue by ILoveYC** [pytorch/pytorch#31987]
## ❓ Questions and Help

Hi,

As we all know, when executing loss.backward(create_graph=True, retain_graph=True). The gradient of leaf nodes (say a Tensor lf) are computed and stored in lf.grad. The question is:

Is the tensor " lf.grad " it self becoming part of the computation graph after backward operation?

This is meaningful since if the answer is true, we can use lf.grad to compute higher-order derivative via:

`xx = lf.grad;
xx.requires_grad_();
grad = torch.autograd.grad(xx, [variables in xx])
`

Thanks!

**Comment by albanD** [comment]
Hi,

As mentioned in the template, we use github issue only for bug reports or features. Please use the forum for questions: https://discuss.pytorch.org/

For you question, yes that is what the `create_graph=True` argument is doing. And you should not set `xx.requires_grad_()` explicitly. It should already require gradients.
If you have followup questions, please use the forum.
